test/README.md

```diff
@@ -0,0 +1,45 @@
+Tests Info
+==========
+
+This area of the repo contains tests that can be run to check the `openshift-applier` operation.
+
+## Requirements
+- An execution environment where the `openshift-applier` can successfully operate
+  - Consider using the [openshift-applier docker image](https://hub.docker.com/r/redhatcop/openshift-applier/)
+- A target OpenShift environment
+  - *Note:* The `openshift-applier` needs access to the `oc` command
+
+## Dependencies
+
+- Ansible 2.5 or later
+- Operational OpenShift Cluster
+- `oc` client
+
+# Running Tests
+
+Each "test case" is built out as an inventory with corresponding files (if applicable) in this directory. In general, and unless otherwise noted, the tests can be executed with the following command (from the repo's top level):
+
+> ansible-playbook playbooks/openshift-cluster-seed.yml -i <path-to-inventory>
+
+List of example test runs:
+
+```
+ansible-playbook playbooks/openshift-cluster-seed.yml -i test/inventories/multi-files-dir
+ansible-playbook playbooks/openshift-cluster-seed.yml -i test/inventories/multi-params-dir
+ansible-playbook playbooks/openshift-cluster-seed.yml -i test/inventories/params-from-file
+ansible-playbook playbooks/openshift-cluster-seed.yml -i test/inventories/params-from-vars
+ansible-playbook playbooks/openshift-cluster-seed.yml -i test/inventories/pre-post-steps
```

Comment:
This test fails to execute with:
```
TASK [openshift-applier : Run ansible-galaxy to pull in dependency roles] ****************************************************************************************************************************************************************************************************************
failed: [localhost] (item=/home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml) => {"changed": true, "cmd": ["ansible-galaxy", "install", "-r", "/home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml", "-p", "/tmp/ansible.GglKOe/"], "delta": "0:00:00.424059", "end": "2018-08-09 15:00:18.452520", "item": "/home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml", "msg": "non-zero return code", "rc": 1, "start": "2018-08-09 15:00:18.028461", "stderr": "ERROR! Unable to open /home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml: [Errno 2] No such file or directory: u'/home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml'", "stderr_lines": ["ERROR! Unable to open /home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml: [Errno 2] No such file or directory: u'/home/esauer/src/openshift-applier/test/inventories/pre-post-steps/../../files/galaxy/requirements.yml'"], "stdout": "", "stdout_lines": []}
```

Reply:
Now I'm getting:
```TASK [/tmp/ansible.Es8tQE/casl-ansible/roles/openshift-labels : Apply label openshift-applier=true to object route] **********************************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["oc", "label", "--overwrite", "route", "route1-test", "openshift-applier=true"], "delta": "0:00:00.500971", "end": "2018-08-09 16:06:55.153388", "msg": "non-zero return code", "rc": 1, "start": "2018-08-09 16:06:54.652417", "stderr": "Error from server (NotFound): routes.route.openshift.io \"route1-test\" not found", "stderr_lines": ["Error from server (NotFound): routes.route.openshift.io \"route1-test\" not found"], "stdout": "", "stdout_lines": []}

RUNNING HANDLER [openshift-applier : Clean up temporary dependency dir] ******************************************************************************************************************************************************************************************************************
	to retry, use: --limit @/home/esauer/src/openshift-applier/playbooks/openshift-cluster-seed.retry

PLAY RECAP *******************************************************************************************************************************************************************************************************************************************************************************
localhost                  : ok=34   changed=4    unreachable=0    failed=1   
```

Reply:
@etsauer congratulation, that's an actual breakage in `openshift-applier`, so the test is valid, but we need to fix openshift-applier. CI is already paying off. :-)

Reply:
@etsauer this bug has now been fixed (in the `master` branch) - do you mind re-testing?